docs/getting-started/continuous_integration.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: default
+title: Continuous Integration
+parent: Getting started
+nav_order: 5
+permalink: /getting-started/continuous-integration/
+---
+
+# Continuous Integration
+
+OSS-Fuzz offers Continuous Integration(CI) support for projects hosted on
+GitHub. This service is called CIFuzz and can be used to run OSS-Fuzz fuzz targets
+on your project when a pull request is submitted. It has the advantage of
+detecting bugs before they are integrated into your project. This insures that
+less bugs are committed meaning less active security vulnerabilities.
+
+## How it Works
+
+CIFuzz works by checking out a repository at the head of a pull request. The
+projects OSS-Fuzz fuzz targets are built and run for
+a definite amount of time (default is 10 minutes). If a bug is found, the
+stack trace as well as the test case are uploaded for the user. If a bug is
+not found the test passes with a green check.
+
+## Requirements
+1. Integrated in OSS-Fuzz.
+2. At-least one build-able fuzz target.
```

Comment:
I'd skip these. If there are no buildable fuzz targets, that means that project is not integrated with OSS-Fuzz

Reply:
Removed.